aptos-move/framework/move-stdlib/sources/option.move

```diff
@@ -189,6 +189,7 @@ module std::option {
     }
     spec swap_or_fill {
         pragma opaque;
+        aborts_if false;
```

Comment:
Why add this?

Reply:
This is not turning off the abort check. This asserts that this function should not abort.